portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,43 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> commentList;
+
+  @Override
+  public void init() {
+    commentList = new ArrayList<>();
+    commentList.add("Hello Alisa!");
+    commentList.add("Hola");
+    commentList.add("How are you?");
+  }
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String json = convertToJson(commentList);
```

Comment:
Can this be done during init() instead of on each request?

Reply:
Yes, this can be done in init() but convert in the doGet method would be useful when commentList updates due to post requests.

Reply:
Yes, that makes sense. The server should maintain an internal data structure rather than a specific presentation of that data